main.css

```diff
@@ -0,0 +1,43 @@
+h1{
+    font-family: monospace;
+}
+
+h2{
+    font-family: monospace;
+}
+
+body{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.board{
+    position: relative;
+    width: 300px;
+    height: 300px;
+    display: flex;
+    flex-direction: column;
+}
+
+.row{
+    width: 300px;
+    height: 100px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+.cell{
+    width: 90px;
+    height: 90px;
+    border: 1px solid;
+    text-align: center;
+    font-size: 80px;
+}
+
+#clear-btn{
```

Comment:
In general we avoid styling with ids. Better if you add to the element both class for styling and id for js logic